### ISSUE TITLE
NVDA reads irrelevant content before reading showQuickPick content

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#58005]
Repro Steps:-
1. Show any dropdown using vscode.window.showQuickPick API
2. Focus the first item in the dropdown
3. Verify that NVDA reads the content without any extraneous details

OR
1. Install Live Share extension for VSCode
2. Press customized keyboard shortcut for "Sign In using browser(Sign In)" to open the Sign In window in browser.
3. Open the link shared by host to join collaboration session in browser.
4. Then navigate to "share icon" and press enter
5. Verify how NVDA is reading the combo box
6. Navigate to "Collaborators icon" and press enter
7. Verify how NVDA is reading the combo box

Actual Result:-
When user focuses first item in the dropdown, NVDA is reading some other content before reading details regarding focused item. NVDA is reading content regarding the opened folder before reading the combo box

Expected Result:-
NVDA should read all the controls in the application correctly. It should not read extra content

User Impact:-
Disabled users will face difficulties while using the application if Screen Reader doesn't read the content correctly

MAS Reference:-
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

OS: Windows 10 Enterprise 17134.228

**Comment by chrmarti** [comment]
@cait1inhennessy It sometimes reads from the currently open file in between two comboboxes. Because the first disappears before the next in shown, there is time for the currently opened file to briefly get focus.
